CHANGELOG.rst

```diff
@@ -3,6 +3,11 @@ UNRELEASED
 
 * Add new unit: "Stokes" (``St``).
 * Use ``TypeCheckingSupport`` from ``oop-ext 1.1``.
+* Add ``cubic feet per day per psi`` (``ft3/psi.d``) unit to ``productivity index`` category.
+* Add ``calories/metre hour degree Celsius`` (``cal/m.h.degC``) unit to ``thermal conductivity`` category.
```

Comment:
Are we using _metre_ (british) or _meter_ (u.s. english)?

Reply:
Hmm, taking a quick look at posc.py we use both writings, but `*metre` is more common.

Reply:
Hmm taking a quick look in posc.py we use both forms, but _*metre_ is more common.